src/models/task/model.py

```diff
@@ -0,0 +1,147 @@
+class TaskOrientedChatbot:
+    def __init__(self):
+        self.done = False
+        self.collected_data = self._init_data()
+        self.stage = 0
+        "possible stages:\n"
+        "        0. ask to confirm task start\n"
+        "        1. confirm start\n"
+        "        2. ask for information\n"
+        "           provide answer\n"
+        "        3. confirm end\n"
+        "           ask for rechecking\n"
+        "        4. confirm rechecking\n"
+        "        5. recheck information\n"
+        self.current_field = None
+
+
+    def _is_completed(self):
+        return self.done
+
```

Comment:
The convention is to only start the names of private fields/methods with "_".
Here, `_is_completed` is a public getter for private field `self.done`, so the names should be:
- `_is_completed` -> `is_completed`
- `self.done` -> `self._done`

Reply:
This also apply to the rest of the class fields/methods :)